_clients/k8s-operator.md

```diff
@@ -0,0 +1,146 @@
+
+If this is your first time running Kubernetes and you intend to run through these instructions on your laptop, make sure that you have the following installed: 
+
+- [Kubernetes](https://kubernetes.io/docs/tasks/tools/) 
+- [Docker](https://docs.docker.com/engine/install/)
+- [minikube](https://minikube.sigs.k8s.io/docs/start/)
+
+Before running through the installation steps, make sure that you have a Kubernetes environment running locally. When using minikube, open a new terminal window and enter `minikube start`. Kubernetes will now use a containerized minikube cluster with a namespace called `default`.
+
+Then install the OpenSearch Kubernetes Operator using the following steps:
+
+1. In your preferred directory, clone the [OpenSearch Kubernetes Operator repo](https://github.com/Opster/opensearch-k8s-operator). Navigate into repo's directory using `cd`.
+2. Go to the `opensearch-operator` folder.
+3. Enter `make build manifests`.
+4. Start a Kubernetes cluster. When using minikube, open a new terminal window and enter `minikube start`. Kubernetes will now use a containerized minikube cluster with a namespace called `default`. Make sure that `~/.kube/config` points to the cluster.
+
+```yml
+apiVersion: v1
+clusters:
+- cluster:
+    certificate-authority: /Users/naarcha/.minikube/ca.crt
+    extensions:
+    - extension:
+        last-update: Mon, 29 Aug 2022 10:11:47 CDT
+        provider: minikube.sigs.k8s.io
+        version: v1.26.1
+      name: cluster_info
+    server: https://127.0.0.1:61661
+  name: minikube
+contexts:
+- context:
+    cluster: minikube
+    extensions:
+    - extension:
+        last-update: Mon, 29 Aug 2022 10:11:47 CDT
+        provider: minikube.sigs.k8s.io
+        version: v1.26.1
+      name: context_info
+    namespace: default
+    user: minikube
+  name: minikube
+current-context: minikube
+kind: Config
+preferences: {}
+users:
+- name: minikube
+  user:
+    client-certificate: /Users/naarcha/.minikube/profiles/minikube/client.crt
+    client-key: /Users/naarcha/.minikube/profiles/minikube/client.key
+```    
+   
+5. Enter `make install` to create the CustomResourceDefinition that runs in your Kubernetes cluster. 
+6. Start the OpenSearch Kubernetes Operator. Enter `make run`. 
+
+## Verify Kubernetes deployment
+
+To ensure that Kubernetes recognizes the OpenSearch Kubernetes Operator as a namespace, enter `k get ns | grep opensearch`. Both `opensearch` and `opensearch-operator-system` should appear as `Active`.
+
+With the operator active, use `k get pod -n opensearch-operator-system` to make sure that the operator's pods are running. 
+
+```
+NAME                                              READY   STATUS   RESTARTS   AGE
+opensearch-operator-controller-manager-<pod-id>   2/2     Running  0          25m
+```
+
+With the Kubernetes cluster running, you can now run OpenSearch inside the cluster.
+
+## Deploy a new OpenSearch cluster
```

Comment:
it should be clearly noted that these are purely examples and are not production ready (using demo TLS certificates, etc.)